dual_net.py

```diff
@@ -52,7 +59,38 @@
 
 # See www.moderndescartes.com/essays/shuffle_viz for discussion on sizing
 flags.DEFINE_integer('shuffle_buffer_size', 20000,
-                     'Size of buffer used to shuffle train examples')
+                     'Size of buffer used to shuffle train examples.')
+
+flags.DEFINE_bool('use_tpu', False, 'Whether to use TPU for training.')
+
+flags.DEFINE_string(
+    'tpu_name', None,
+    'The Cloud TPU to use for training. This should be either the name used'
+    'when creating the Cloud TPU, or a grpc://ip.address.of.tpu:8470 url.')
+
+flags.register_multi_flags_validator(
+    ['use_tpu', 'tpu_name'],
+    lambda flags: bool(flags['use_tpu']) == bool(flags['tpu_name']),
+    'If use_tpu is set, tpu_name must also be set.')
+
+flags.DEFINE_integer(
+    'iterations_per_loop', 100,
```

Comment:
does this mean metrics every 100 * BATCH_SIZE positions?  Should train_batch_size change with use_tpu?

Reply:
(maybe just update the comment in train_batch_size's usage instructions to suggest a better default value for tpus)

Reply:
fixed